db/migrate/20191118150004_create_activities.rb

```diff
@@ -0,0 +1,22 @@
+class CreateActivities < ActiveRecord::Migration[6.0]
+  def change
+    create_table :activities, id: :uuid do |t|
+      t.string :identifier
+      t.string :sector
+      t.string :title
+      t.text :description
+      t.string :status
+      t.date :planned_start_date
+      t.date :planned_end_date
+      t.date :actual_start_date
+      t.date :actual_end_date
+      t.string :recipient_region, default: "998"
```

Comment:
These default options from the tickets were intended (my bad) to be default form options rather than database defaults. I think those default values wouldn't make sense when being filled out from the perspective of a grant level activity, hopefully we can use form helpers to provide sensible defaults dynamically to help the user.

I think these 3 defaults can be taken out 👍